hub/util/auto.py

```diff
@@ -50,3 +51,56 @@ def get_most_common_extension(
     compression = most_common_extension[0].split(".")[1]
 
     return compression
+
+
+def ingestion_summary(local_path: str, skipped_files: list, ingested_file_count: int):
```

Comment:
could you add tests for this?

Reply:
done, the function doesn't return anything so I suppose using the `stdout` method to store the output might be our only shot

https://github.com/activeloopai/Hub/blob/28b97cef584a98c83f2439f388c99882af9e538a/hub/util/tests/test_auto.py